CODEOWNERS

```diff
@@ -1,9 +1,8 @@
-/plugin/ @mattermost/toolkit
-
+/plugin/ @mattermost/integrations
```

Comment:
I'm seeing an error with the CODEOWNERS file with this change:

> Unknown owner: make sure the team `@mattermost/integrations exists`, is publicly visible, and has write access to the repository

@lieut-data Let's remove the code owner for now, and our team can revisit with https://mattermost.atlassian.net/browse/MM-52111

![CleanShot 2023-04-11 at 15 32 40](https://user-images.githubusercontent.com/6913320/231269511-c42dea11-18c5-4707-a9a3-1545a70710a0.png)